staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -1056,7 +1056,7 @@ func getResourceNamesForGroup(apiPrefix string, apiGroupInfo *APIGroupInfo, path
 // Eventf creates an event with the API server as source, either in default namespace against default namespace, or
 // if POD_NAME/NAMESPACE are set against that pod.
 func (s *GenericAPIServer) Eventf(eventType, reason, messageFmt string, args ...interface{}) {
-	t := metav1.Time{Time: time.Now()}
+	t := metav1.MicroTime{Time: time.Now()}
```

Comment:
Is this method (`Eventf`) used only for generating shutdown related events ?
Asking because Increasing precision for all events will also increase the storage size.